src/index.html

```diff
@@ -4,9 +4,352 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home">
+        <img src="images/logo.png"
+             alt="moyo-logo"
+             class="header__logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple"
+               class="nav__link is-active">
+               Apple
+            </a>
```

Comment:
Fix all cases please
```suggestion
            <a
              href="#apple"
              class="nav__link is-active"
            >
               Apple
            </a>
```